oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStore.java

```diff
@@ -152,7 +152,7 @@ public final class DocumentNodeStore
 
     private static final Logger LOG = LoggerFactory.getLogger(DocumentNodeStore.class);
 
-    private static final PerfLogger PERFLOG = new PerfLogger(
+    static final PerfLogger PERFLOG = new PerfLogger(
```

Comment:
Quite liked the approach of the the other two cases where a new static `configurePerfLogger` is introduced. Avoids requiring this static to be made open.

Reply:
Done